openadmet/toolkit/tests/chemoinformatics/test_cheminf.py

```diff
@@ -64,3 +72,19 @@ def test_reaction_smart_retro(amide_dataframe):
     df = retro.run(amide_dataframe, smiles_column="SMILES")
     assert all(pd.notna(df["amide_coupling_amine_inchikey"]))
     assert all(pd.notna(df["amide_coupling_carboxylic_acid_inchikey"]))
+
+def test_calculate_pac50():
+    ic50_1 = calculate_pac50(1, input_unit_str="uM")
+    ic50_2 = calculate_pac50(10, input_unit_str="nM")
+    ic50_3 = calculate_pac50(3e-8, input_unit_str="M")
+    assert ic50_1 == 6.0
+    assert ic50_2 == 8.0
+    assert round(ic50_3, 1) == 7.5
+
+def test_pac50_to_ki():
+    ki = pac50_to_ki(9.0)
+    assert ki == 1e-9 * unit.molar
+
+def test_ki_to_dg():
+    dg = ki_to_dg(ki=100, input_unit_str="nM")
+    assert round(dg, 1) == -40.0 * unit.kilojoule_per_mole
```

Comment:
Same here not to use `round` in testing. Use numpy testing functions e.g `np.assert_allclose()`